handlers/misttriggers/triggers_test.go

```diff
@@ -89,7 +89,7 @@ func TestRecordingCompleted(t *testing.T) {
 		require.NotNil(t, message.Success)
 		require.True(t, *message.Success)
 		require.GreaterOrEqual(t, message.Timestamp, testStartTime)
-		require.Less(t, message.Timestamp, testStartTime+2)
+		require.Less(t, message.Timestamp, testStartTime+100)
```

Comment:
Nothing changed here, but the timing was a bit too tight on CI and so I was seeing some failures